PYME/recipes/base.py

```diff
@@ -733,7 +733,7 @@ def update_from_yaml(self, data):
             with open(data) as f:
                 data = f.read()
     
-        l = yaml.load(data)
+        l = yaml.load(data, Loader=yaml.FullLoader)
```

Comment:
Better to change to `load_safe` - if we are doing an unsafe load we want to be warned about it.